src/Data.Json.JSemantic.purs

```diff
@@ -0,0 +1,105 @@
+module Data.Json.JSemantic
+  ( JSemantic(..)
+  , toSemantic
+  ) where
+
+import Data.Argonaut.JCursor(JsonPrim(..), runJsonPrim)
+import Data.Maybe
+import Data.String.Regex (RegexFlags(..), regex, test)
+import Data.String
+import Data.Foldable
+import qualified Data.Array as A
+import qualified Data.Array.Unsafe as AU
+import Data.Tuple
+import qualified Data.Date as Date
+import Math
+
+
+data JSemantic = Integral   Number
```

Comment:
This looks good and means we don't have to pass Json to the styler, which is nice. :+1: